composer.json

```diff
@@ -32,7 +32,7 @@
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
         "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/symfony": "^2.8.7, !=2.8.10 || ^3.0",
```

Comment:
Understood, anyway symfony3 support is not the main point of this PR, I exended version constraint just to avoid additional commit/PR ;) On the other hand, just as mentioned earlier - with stable versions of deps it shouldn't resolve to 3.x, but 2.8 instead.